test/models/repository_test.rb

```diff
@@ -441,33 +441,71 @@ def test_docker_repository_in_content_view
       assert view_repo.valid?
     end
 
-    def test_docker_repository_docker_upstream_name_format
+    test_attributes :pid => '3360aab2-74f3-4f6e-a083-46498ceacad2'
+    def test_create_docker_with_valid_name
       @repo.unprotected = true
       @repo.content_type = 'docker'
       @repo.download_policy = nil
-      valid = %w( valid
-                  abc/valid
-                  thisisareallylongbutstillvalidname
-                  soisthis/thisisareallylongbutstillvalidname
-                  single/slash
-                  multiple/slash/es abc/def/valid
-                  )
-      valid << 'a' * 255
-      valid.each do |name|
+      @repo.docker_upstream_name = 'busybox'
+      valid_name_list.each do |name|
+        @repo.name = name
+        assert @repo.valid?, "Validation failed for create with valid name '#{name}' length: #{name.length}"
+      end
+    end
+
+    test_attributes :pid => '742a2118-0ab2-4e63-b978-88fe9f52c034'
+    def test_create_docker_with_valid_upstream_name
+      @repo.unprotected = true
+      @repo.content_type = 'docker'
```

Comment:
Seems like we need a new fixture which is a docker repo since there seems to be quite a bit of duplication in this file setting up a docker repo: https://github.com/Katello/katello/blob/master/test/fixtures/models/katello_repositories.yml

Reply:
@jturel many thanks for your review, why we need a  fixture as any way we have to create that docker entity, (The test is not really creating but only validate on the created object)

Reply:
Some months have gone by ... I don't think we need, but I think we could benefit from FactoryBot?